package.json

```diff
@@ -4,6 +4,7 @@
   "description": "REST storage and service dispatcher",
   "main": "server.js",
   "scripts": {
+    "start": "servisor",
```

Comment:
`npm start` will run `service-runner` which will read RESTBase's config and start it.